pymobiledevice3/tunneld.py

```diff
@@ -356,6 +374,11 @@ async def clear_tunnels() -> fastapi.Response:
             self._tunneld_core.clear()
             return fastapi.Response(status_code=200, content='Cleared tunnels...')
 
+        @self._app.get('/cancel-tunnel')
```

Comment:
Since `/clear` is not called `/clear-tunnels` and `/shutdown` isn't `/shutdown-tunneld`, lets keep the same semantics.
```suggestion
        @self._app.get('/cancel')
```